app/views/content/is-teaching-right-for-me/teaching-internship-providers.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Teaching internships
 heading: "Experience teaching with an internship"
-subcategory: Experience life in the classroom
+subcategory: Experience the classroom
```

Comment:
I'm not sure if this subcategory quite fits with the other ones now as it starts with a verb rather than a noun? However, I don't feel too strongly about this!